### DISCUSSION TITLE
guide to install msdos

### DISCUSSION THREAD

**Discussion by Mad-Soft** [dosemu2/dosemu2#1598] [General]
I see in man pages the possibility with dosemu.conf to boot with original disk1.img of ms-dos 6.2.
But how install? if installer can't indentify start partition 
Do I have to create a C image like dosbox? hdd_c.img
Or it can be installed in a directory?

**Comment by stsp** [comment]
You can install or copy into a directory.
Or you can use an image.
In `$_hdimage` you can refer to images
and to directories the same way.
But dosemu2 is not about installing anything.
If you don't have a special need to boot
any particular dos, then don't install anything
at all.
I.e. any way you want, everything is supported.
